src/mmda/predictors/heuristic_predictors/figure_table_predictors.py

```diff
@@ -106,7 +104,7 @@ def generate_map_of_layout_to_tokens(
         """
         for page in vila_dict.keys():
             for span in vila_dict[page]:
-                for layout_span in dict_of_pages_layoutparser.get(page):
+                for layout_span in dict_of_pages_layoutparser[page]:
```

Comment:
This works right now but depends on the method being called with a defaultdict(list), which seems like a regression waiting to happen.

Can we do

```
for layout_span in dict_of_pages_layoutparser.get(page, []):
```

instead, for clarity and future proofing?